views/extras.tmpl

```diff
@@ -205,27 +202,29 @@
                 $(newRowHtml).insertBefore(expTableRows.first())
                 setTimeElements()
             }
+            var ex = newBlock.extra
             if ($("#bsubsidy_total").text() !== "") {
-                $("#bsubsidy_total").html(humanize.decimalParts(b.next_block_subsidy.total/100000000, false, 8))
-                $("#bsubsidy_pow").html(humanize.decimalParts(b.next_block_subsidy.pow/100000000, false, 8))
-                $("#bsubsidy_pos").html(humanize.decimalParts((b.next_block_subsidy.pos/500000000), false, 8))
-                $("#bsubsidy_dev").html(humanize.decimalParts(b.next_block_subsidy.developer/100000000, false, 8))
-                $("#coin_supply").html(humanize.decimalParts(b.coin_supply/100000000, true, 8))
-                $("#blocksdiff").html(humanize.decimalParts(b.sdiff, false, 8))
-                $("#window_block_index").text(newBlock.stake.window_block_index)
-                $("#pos-window-progess-bar").css({ width: (newBlock.stake.window_block_index/144)*100 + "%" })
+                $("#difficulty").html(humanize.decimalParts(ex.difficulty/100000000, false, 8))
+                $("#bsubsidy_total").html(humanize.decimalParts(ex.subsidy.total/100000000, false, 8))
+                $("#bsubsidy_pow").html(humanize.decimalParts(ex.subsidy.pow/100000000, false, 8))
+                $("#bsubsidy_pos").html(humanize.decimalParts((ex.subsidy.pos/500000000), false, 8))
```

Comment:
I'm OK with assuming 5 votes per block as it's that way on all networks, but perhaps a comment here.